README.md

```diff
@@ -55,11 +55,21 @@ fastify.register(require('fastify-circuit-breaker'), {
   threshold: 3, // default 5
   timeout: 5000, // default 10000
   resetTimeout: 5000 // default 10000
+  onCircuitOpen: (req, reply) => {
+    reply.statusCode = 500
+    throw new Error('a custom error')
+  },
+  onTimeout: (req, reply) => {
+    reply.statusCode = 504
+    return 'timed out'
+  }
```

Comment:
@delvedor do you think we should support async/await for those hooks?

Reply:
Yes, it would be better :)

Reply:
@ryhinchey could you make it so then?

Reply:
@mcollina Yep! Will do

Reply:
@mcollina @delvedor I just updated to support async/await for the hooks